src/20231213_AaveV3Gnosis_UpdateGNORiskParametersOnAaveV3GnosisPool/UpdateGNORiskParametersOnAaveV3GnosisPool.md

```diff
@@ -0,0 +1,48 @@
+---
+title: "Update GNO Risk Parameters on Aave V3 Gnosis Pool"
```

Comment:
perhaps better "Enable GNO borrowing"

Reply:
As there are some other params updated (e.g. borrow cap, RF), I would say better `Update GHO parameters on Aave V3 Gnosis Pool`, general enough

Reply:
Same to be applied on the payload file itself, not only AIP